documentation.Rmd

```diff
@@ -141,12 +144,13 @@ When linking to external resources either include the full url inline with `<>`,
 
 Text that contains valid R code should be marked as such using backticks. This includes:
 
-* Function names, which should be followed by `()`, e.g. `tibble()`.
 * Function arguments, e.g. `na.rm`.
 * Values, e.g. `TRUE`, `FALSE`, `NA`, `NaN`, `...`, `NULL`
 * Literal R code, e.g. `mean(x, na.rm = TRUE)`
 * Class names, e.g. "a tibble will have class `tbl_df` ..."
 
+You can use code syntax for function names, but consider if it would be better to use a cross-link instead.
```

Comment:
```suggestion
You can use code syntax for function names, like `tibble()`, but consider if it would be better to use a cross-link instead, like `[tibble()]`.
```

I don't think it hurts to be heavy on the examples here

Reply:
partially-related aside, I like the recommendation "link the first reference, no need on subsequent references", but there's a tension where the order we write in roxygen comments is not necessarily the order in the .Rd file, so it's hard to make the programmer-facing link behavior match the user-facing version.

That might be an {roxygen2} FR...